lib/cadet/jobs/autograder/grading_job.ex

```diff
@@ -88,6 +92,14 @@ defmodule Cadet.Autograder.GradingJob do
     end
   end
 
+  def assessment_requires_plagiarism_check(assessment = %Assessment{}) do
+    Question
+    |> where(assessment_id: ^assessment.id)
+    |> select([:type])
```

Comment:
I think it's better to have this as `|> select([q], q.type)` and later on `|> Enum.any?(& &1 == :programming)` to save on memory not having to load the whole struct.